src/NuGet.Indexing/ServiceHelpers.cs

```diff
@@ -51,29 +52,28 @@ public static void WriteResponse(IOwinContext context, ClientException e)
         }
 
 
-        public static void WriteResponse(IOwinContext context, Exception e)
+        public static void WriteResponse(IOwinContext context, Exception e, FrameworkLogger logger)
```

Comment:
In general it seems this file seems tied to running in a web service, maybe it should be moved to the Search project?  The one exception that I can see is the TraceException method which seems like a) a good candidate for the Indexing project and b) useful for addressing @maartenba's comment about stack trace.